.github/workflows/tests.yml

```diff
@@ -153,16 +148,24 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-pip-
 
+      - name: Install Build Dependencies (3.13)
+        if: matrix.python-version == '3.13'
+        run: |
+          pip install -U pip
+          pip install -U "setuptools<69" wheel twine
+          # Remove this hack once 'cffi' has a release supporting Python 3.13.
+          pip install -e "git+https://github.com/python-cffi/cffi.git#egg=cffi"
```

Comment:
There is now an alpha release with Python 3.13 support: https://pypi.org/project/cffi/1.17.0rc1/ we could try to use it.

Reply:
Good catch, this works.